src/main/java/uk/ac/ebi/pride/gui/task/FileScanAndValidationTask.java

```diff
@@ -911,7 +911,11 @@ private List<DataFile> runMzIdentMLPeakFilesValidation(List<DataFile> dataFiles)
                     for (DataFile spectraFile : dataFiles) {
                         String spectraFileName = FileUtil.getDecompressedFileName(spectraFile.getFile());
                         if (peakListFileName.equalsIgnoreCase(spectraFileName)) {
-                            if (spectraFile.getFileType() != ProjectFileType.PEAK) {
+                            if (spectraFile.getFileType() != ProjectFileType.PEAK &&
+                                    !(spectraFile.getFileType() == ProjectFileType.RAW &&
+                                            (spectraFile.getFileFormat() != MassSpecFileFormat.MZML ||
+                                                    (spectraFile.getFileFormat() != MassSpecFileFormat.INDEXED_MZML))))
```

Comment:
This will allow if the filetype is RAW and fileFormat is MZTAB/MZIDENTML or others. If the intent is to allow only MZML/INDEXED_MZML as RAW, the condition should be !(spectraFile.getFileType() == ProjectFileType.RAW &&
                                            (spectraFile.getFileFormat().equals(MassSpecFileFormat.MZML) ||
                                                    (spectraFile.getFileFormat().equals(MassSpecFileFormat.INDEXED_MZML)))
                                                    
                                                    
  And also check u want to allow this two formats ?
  INDEXED_MZML("mzml", true, ProjectFileType.PEAK),
    MZML("mzml", true, ProjectFileType.PEAK)